scripts/setup_colab.sh

```diff
@@ -217,8 +208,8 @@ cp "${_arg_mjkey}" "${HOME}/.mujoco/mjkey.txt"
   MUJOCO_150="$(mktemp -d)/mujoco_150"
   git clone --branch 1.50.1.1 --depth 1 https://github.com/openai/mujoco-py.git $MUJOCO_150
```

Comment:
I think this is still using old version of mujoco_py.

Reply:
Great catch. Missed it probably because everywhere else it was called mjpro150. Will fix this